model/patch_lifecycle.go

```diff
@@ -438,9 +438,61 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 		return nil, errors.WithStack(err)
 	}
 
+	if p.IsParent() {
+		//finalize child patches that have a parentStatus = ""
+		for _, childPatch := range p.Triggers.ChildPatches {
+			err = finalizeChildPatch(ctx, childPatch, requester, githubOauthToken)
+			grip.Error(message.WrapError(err, message.Fields{
+				"message":       "unable to finalize child patch",
+				"childPatch_id": childPatch,
+				"patch_id":      p.Id,
+				"variants":      p.BuildVariants,
+				"tasks":         p.Tasks,
+				"variant_tasks": p.VariantsTasks,
+				"alias":         p.Alias,
+			}))
+			return nil, err
```

Comment:
I could hear continuing on error _here_

Reply:
I could hear it too, but is there any benefit to using compute time doing it if the resulting patch will be in a state that's not really viewable by users?

Reply:
that's a sound argument 😄